### ISSUE TITLE
Share With Everyone?

### ISSUE THREAD

**Issue by Mike-Bell** [microsoftgraph/microsoft-graph-docs-contrib#8337]
I'd like to use the API to share a folder I've created with everyone in my ODFB org. Like this:

![image](https://cloud.githubusercontent.com/assets/8780033/12850279/e2788020-cbf1-11e5-97dc-da1b7f1d97af.png)

After looking through the documentation, I see no clear way to do this.

I'm not sure if this documentation is relevant: http://graph.microsoft.io/en-us/docs/api-reference/v1.0/api/permission_get and to be honest, I find that documentation very confusing. An example request is `GET /drive/items/<id>/permissions/<id>`. I assume the first ID is an item ID but what's the second ID? A permissions ID? How would I even know that? Or is it a user ID that has permissions on the item? It would be nice if this documentation were clearer. And I still don't even know if this gets me what I want.

I tried a little reverse engineering by manually sharing a folder with everyone, and seeing if I could find some flag on the folder through the API that had been flipped, but I wasn't  successful. I tried doing a `GET /permissions` on an item that I had 'Share With Everyone', but the permissions object comes back as a blank array (???). Not very helpful.

As an alternative, does this functionality work in the older OneDrive API?

tldr I'd like to share a folder with everyone in my org, and permissions documentation is confusing.
